scripts/swap_safe_owner.py

```diff
@@ -85,7 +85,7 @@ def load_contract(ctype: ContractType) -> ContractType:
         current_owner_crypto = EthereumCrypto(
             private_key_path=args.current_owner_private_key_path, password=args.password
         )
-        owner_cryptos: list[EthereumCrypto] = [current_owner_crypto]
+        owner_cryptos = [current_owner_crypto]
```

Comment:
Can you explain?

Reply:
Okay, saw https://github.com/valory-xyz/trader-quickstart/pull/219#discussion_r1478481209.